ansible/tasks/base.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: "System deps"
+  become: yes
+  apt:
+    name:
+      - apt-transport-https
+      - ca-certificates
+      - curl
+      - gnupg2
+      - pass
```

Comment:
I thought `pass` was a CLI to manage passwords using `gnupg2`. I am curious, how come we need it?

Reply:
Yes good question, this list was taken from some prerequisites for Ansible which sometimes does some funky stuff with passwords. Admittedly we may get away without it as we don't really use any of those kind of features.